src/components/Plot/Plot.js

```diff
@@ -132,7 +134,7 @@ export const Plot = ({
         placement: 'top',
         title: (
           <>
-            <Typography>{item.name}</Typography>
+            <Typography>{item ? item.name : 'shoveled'}</Typography>
```

Comment:
i think this logic can be improved in general with the update from your other comment to add something actually signaling this plot was shoveled. i do agree it's not great as the general fallback! 

re: strings, no problem moving this one, but what do you think about adopting something like [i18next](https://www.i18next.com/) in the near future? we don't need to actually localize strings (yet), but we would be prepared to do so, while also keeping them separated. this would also give us an interface for strings that use interpolation, or dynamic count values (the `count` arg is particularly powerful). happy to share some examples if you've never worked with i18next before!

Reply:
>what do you think about adopting something like i18next in the near future?

I'm into it! I've used i18next lightly on a few projects and I think it would be good to use as a matter of best practice at the very least. That said, I don't feel a rush to integrate it, as it doesn't solve a problem that we currently have. I'll leave the timeline and strategy up to you for doing this, but I'll support you with whatever you need to get it done! 👍

Reply:
makes sense to me!